rosco-web/config/rosco.yml

```diff
@@ -133,12 +133,12 @@ azure:
     baseImages:
     - baseImage:
         id: ubuntu
-        shortDescription: v14.04
-        detailedDescription: Ubuntu Server 14.04.4-LTS
+        shortDescription: v14.05
```

Comment:
Understood it looks wrong, but when we checked VM skus using our Cli this was correct.